action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
   files:
     description: "List of bake definition files"
     required: true
+  working-directory:
```

Comment:
let's name it `workdir`

Reply:
I chose `working-directory` because its the term [GitHub Action uses in their API](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrun). But I can change to `workdir` if you still think it's better.

Reply:
Yes, I prefer not to confuse it with the core API and also to keep it a single word.

Reply:
Done